modules/lvms-troubleshooting-recovering-from-missing-lvm-storage-or-operator-components.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in the following assemblies: 
+//
+// storage/troubleshooting-storage.adoc
+
+:_content-type: PROCEDURE
+[id="recovering-from-missing-lvm-storage-or-operator-components_{context}"]
+= Recovering from missing LVM Storage or operator components
+
+Sometimes the cluster is stuck in a state where it has lost the volume group. If you encounter a storage class "not found" error, check the `LVMCluster` resource and ensure that all the LVM Storage (LVMS) pods are running. You can create an `LVMCluster` resource if it doesn't exist.
```

Comment:
I didn't quite understand this sentence: `Sometimes the cluster is stuck in a state where it has lost the volume group`. Can we maybe delete it?